metaflow/plugins/argo/argo_client.py

```diff
@@ -119,7 +118,7 @@ def trigger_workflow_template(self, name, parameters={}):
     def schedule_workflow_template(self, name, schedule=None, timezone=None):
```

Comment:
what if schedule is None and the schedule exists in the cluster? Does this client delete the schedule?

Reply:
ah you always create schedule, and set `suspend=None` if `schedule==None`... why not delete the `CronWorkflow` instead?